### PR TITLE
Change UI for Statuses

### LINKED ISSUE #109: Change UI of `InterviewStatus`
Currently `InterviewStatus` is displayed as a text on the `PersonCard`. 
Let's change it to a highlighted box for better visibility and better UI experience.

### PR DESCRIPTION
**Pull request by tiewweijian**
Currently, `ApplicationStatus` and `InterviewStatus` are displayed as text in `PersonCard`. 
In this iteration, we have created a new `FlowPane`, and also created new methods to dynamically change the colour of the `Labels` of the `statuses` when they are changed. 

Closes #109 #110

### CODE REVIEW

**Review by lzan98 — APPROVED** [review]
Nice work senpai!

File: src/main/resources/view/CandidateListCard.fxml [hunk 1]
```diff
@@ -27,6 +27,7 @@
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
+      <FlowPane fx:id="statusPane" />
```
Comment (charmainehly) [inline src/main/resources/view/CandidateListCard.fxml:30]: Just a minor suggestion, would it be relevant to include the `prefWrapLength` for the tags! :) - or perhaps you can look into this when the three bottom panels are up as well!

Comment (tiewweijian) [reply src/main/resources/view/CandidateListCard.fxml:30]: Thanks for noting it out !! will make the changes now !

**Review by charmainehly — APPROVED** [review]
LGTM!

**Pull request merged**